content/en/product_analytics/charts/user_retention.md

```diff
@@ -10,44 +10,66 @@ further_reading:
 ## Overview
 Retention Analysis allows you to measure how often users are successfully returning to a page or action. By tracking user retention over time, you can gain insights into overall user satisfaction.
 
-User retention is measured within a given cohort of users that you define. A cohort is a group of users who participate in an initial event, such as clicking a link. A user in the cohort is considered retained if they subsequently complete a return event, such as clicking the same link again or clicking a **Proceed to Payment** button. Only views and actions can act as events.
+User retention is measured within a given cohort of users that you define. A cohort is a group of users who participate in an initial event, such as clicking a link. A user in the cohort is considered retained if they subsequently complete a return event, such as clicking the same link again or clicking a **Proceed to Payment** button. 
 
-The retention graph displays the percentage of users who completed the return event each week.
+Only views and actions can act as events.
+
+The retention graph displays the percentage of users who completed the return event each day during the past week.
+
+{{< img src="product_analytics/retention/pana_retention_overview-2.png" alt="Example Retention Analysis graph" style="width:100%;" >}}
+
+<div class="alert alert-info">
+<strong>Note:</strong> The retention chart displays grey when partial data is available for a given time period or cohort. This is because the time period is not finished and we cannot yet accurately compute the retention rate. 
```

Comment:
```suggestion
<strong>Note:</strong> The retention chart displays grey when partial data is available for a given time period or cohort. This is because the time period is not finished and Datadog cannot yet accurately compute the retention rate. 
```